components/expression_language/syntax.rst

```diff
@@ -127,9 +127,9 @@ Working with Functions
 ----------------------
 
 You can also use registered functions in the expression by using the same
-syntax as PHP and JavaScript. The ExpressionLanguage component comes with one
-function by default: ``constant()``, which will return the value of the PHP
-constant::
+syntax as PHP and JavaScript. The ExpressionLanguage component comes with two
+functions by default. The first one is ``constant()``, which will return the
```

Comment:
Maybe let's use headlines instead of the introduction with "the first..." and "the second"?

Reply:
Updated, as well as your previous feedback!